Week-3/Assingment3.1/Assingment3.1.1/Script.js

```diff
@@ -0,0 +1,19 @@
+function showAlert(alertId) {
+    const bootstrapAlert = document.querySelector("#" + alertId);
+    const collapse = new bootstrap.Collapse(bootstrapAlert); 
+    collapse.show();
+ }
+ 
+ const modalYesBtn = document.querySelector("#contact-modal .btn-primary");
+ modalYesBtn.addEventListener("click", function () {    
```

Comment:
were you asked to use bootstrap classess for css ??

Reply:
yes.its written in the question to add necessary Bootstrap classes